### PR TITLE
Add client side hightlight to pdf theme

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: src/docfx.website.themes/gulpfile.js [hunk 1]
```diff
@@ -12,18 +12,24 @@ var knownOptions = {
 };
 
 var options = minimist(process.argv.slice(2), knownOptions);
-var files = [
-    "fonts/*",
-    "partials/*",
+var preprocessorFiles = [
```
Comment (superyyrrzz) [inline src/docfx.website.themes/gulpfile.js]: >preprocessorFiles [](start = 4, length = 17)

templateFiles?

**Comment by superyyrrzz** [comment]
:shipit:

File: src/docfx.website.themes/pdf.default/bower.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "docfx"
+  ],
+  "license": "MIT",
+  "keywords": [
+    "docfx",
+    "template"
+  ],
+  "homepage": "https://github.com/dotnet/docfx",
+  "moduleType": [],
+  "private": true,
```
Comment (hellosnow) [inline src/docfx.website.themes/pdf.default/bower.json]: >"private": true, [](start = 2, length = 16)

question: why set it as private?

**Comment by hellosnow** [comment]
:shipit:

File: src/docfx.website.themes/pdf.default/bower.json (hunk 2 above)
Comment (vicancy) [inline src/docfx.website.themes/pdf.default/bower.json]: private means our package will not publish to bower https://github.com/bower/spec/blob/master/json.md#private

**Pull request merged**